apps/src/p5lab/poetry/constants.js

```diff
@@ -76,7 +77,7 @@ export const POEMS = {
       'over and over again.',
       'When I go to a new city',
       'with different smells, sounds, and feelings,',
-      'I’ll still remember everything my city had.'
+      "I'll still remember everything my city had."
```

Comment:
I used the "apostrophe" characters instead of the "right single quotation mark" in the Italian poems I added, which seems more common in our translated Italian strings. Updated this to match.

Reply:
It feels odd to my brain that we switch back and forth between these strings being wrapped in single quotes and double quotes, depending on apostrophe use. Are the 'apostrophe' and 'right single quotation mark' rendered similarly? If so, I'd lean towards using the original version so we can keep all the string in single quotes. Thoughts?

Reply:
They do look similar -- I just have no idea how to type a "right single quotation mark" on my keyboard other than to copy/paste it, and I believe an apostrophe is the appropriate character here technically :). I had a mix of them in the Italian poems I was given, so chose apostrophe based on frequency in translations. Looks like [Prettier prefers to use single/double quotes to minimize escapes](https://prettier.io/docs/en/rationale.html#strings), which I think is why I'm mixing them here.

Reply:
This is a real rabbit hole, is the thing that gets typed when I click the key to the right of the semicolon/colon button a single quote or an apostrophe?? I would call it a single quote, but when I look it up in a[ unicode lookup site](https://unicodelookup.com/#'/1) it calls it a apostrophe 🤯

Reply:
I didn't know how to type a slanted apostrophe either and looked it up - it's not easy! On a mac option+shift+] and on windows,  alt+0146. 
I think I agree with Erin that using single quotes for all the strings would be nice for consistency. According to this [article](http://snowball.tartarus.org/texts/apostrophe.html), the slanted apostrophe is actually a single closing quote.

Reply:
Ok, I'd have to update all of the Italian poems to use ’ instead of ' as well, is that what y'all would recommend?  I get that it's kind of odd to go back and forth between single and double quotes in a single context, but I think thats a pretty standard convention for when you want to include single quotes in a string ([prettier](https://prettier.io/docs/en/rationale.html#strings), [rubocop](https://github.com/rubocop/ruby-style-guide#single-quote), or vice versa.

Reply:
Me, trying to decide which way to go here
![eitherway](https://user-images.githubusercontent.com/2959170/192902439-fac7bf62-8b18-4c62-a6f7-a099de06aaa2.gif) 

Based on that prettier link, we should probably go with what you have here originally. I think the fact that it looks out of place is exacerbated because it's part of a list of strings, so it catches the eye. But let's approach each string first as an individual entity, so mixing single-quotes and double-quotes here seems fine. :+1:

Thanks for linking those docs as we reflected on this!

Reply:
Sounds good to me, thanks for thinking it through with me! lol your gif thankfully came through in my email before the edit, amazing choice!